app/src/main/java/org/simple/clinic/home/HomeScreen.kt

```diff
@@ -91,6 +91,7 @@ class HomeScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context
     TabLayoutMediator(homeTabLayout, viewPager) { tab, position ->
       tab.text = resources.getString(tabs[position].title)
     }.attach()
+    viewPager.isUserInputEnabled = false
```

Comment:
Have we confirmed this behaviour change with designers since this might affect users?

Reply:
Good point. Will inform.

Reply:
It's fine for now, please see the conversation at https://simpledotorg.slack.com/archives/CFM321423/p1598347202005400